pkg/api/authn.go

```diff
@@ -55,7 +55,18 @@ func bearerAuthHandler(ctlr *Controller) mux.MiddlewareFunc {
 			}
 			vars := mux.Vars(request)
 			name := vars["name"]
+
+			// we want to bypass auth for mgmt route
```

Comment:
This will not be entirely correct when accessControl is enforced over mgmt config push and data.